rerun_py/rerun_sdk/rerun/_baseclasses.py

```diff
@@ -71,8 +71,21 @@ def __str__(self) -> str:
 
         return s
 
-    def archetype_name(self) -> str:
-        return "rerun.archetypes." + type(self).__name__
+    @classmethod
+    def archetype_name(cls) -> str:
+        return "rerun.archetypes." + cls.__name__
```

Comment:
Just a note: I ran into an issue earlier where subclassing an archetype causes this to have unexpected behavior. We can file as a separate issue though.

Reply:
This wasn't a classmethod until this PR, it was a plain one... that might be why you had a bad surprise earlier?